app/lib/theme/widgets/FileWidget.tsx

```diff
@@ -120,6 +123,7 @@ const FileWidget: React.FC<FileWidgetProps> = ({
           name,
           size,
           type,
+          ...(useFileDate ? { fileDate } : {}),
```

Comment:
I thought we were going to pass back the `created_at` if we weren't using the `useFileDate` here? Or is that just in a different ticket

Reply:
That's a different ticket, it should make more sense now that the other ticket is merged. I'm using `fileDate` to capture the "Date file received" field which might be different than the uploaded date, if that makes sense.